### DISCUSSION TITLE
开源之史系列之二：类比：人类历史上那些重要的暗流涌动 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#13] [Announcements]
# 开源之史系列之二：类比：人类历史上那些重要的暗流涌动 - 开源之道

https://opensourceway.community/posts/history-of-open-source/001-the-undercurrent-of-human-history/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
暗流 的原文是什么?
感觉这说的就是 MEME 模因

- DNA ~ 基因, 决定我们的肉体
- MEME ~ 模因, 决定我们融入在哪个社会
- TEME ~ 技因, 决定我们以何为生, 并依此抵抗 MEME
- 只是从 MEME->TEME 还缺少一环节, 俺命名为 

昧因 ~ Mainium, 获得 TEME 的决定因素,也就是我们得以自学的关键素养;

所以, 普惠社会全体的 暗流 , 其实并不暗, 比如 Wikipedia, 正是几万职业编辑, 几十万积极贡献者, 几百万专题贡献者, 以及几千万积累用户, 才成功逈社会完成这一 MEME 的注入...

开源, 想成为这样的 MEME 还需要更多成功故事, 
至少相关专题电影得有10+, 成为电影背景内容 100+, 电视剧10+, 
独角兽 100+ ...
图书 1000+ ...

为此, 慢慢来, 比较快 ;-)
